docs/proposals/20240807-in-place-updates.md

```diff
@@ -440,15 +441,17 @@ sequenceDiagram
         hook2->>capi: I can update []
         capi->>+hook: Can update [spec.version,<br>clusterConfiguration.kubernetesVersion]?
         hook->>capi: I can update [spec.version,<br>clusterConfiguration.kubernetesVersion]
-        capi->>apiserver: Mark Machine with Update Plan
+        capi->>capi: Decide Update Strategy
+        capi->>apiserver: Mark Machine as pending and set UpToDate condition
```

Comment:
if we want to abbrebiate to this in the doc maybe we can define in the glossary what marking as pending and marking as done mean
those are overloaded terms imo, and here we refer specifically about pendning and done in the context of runtime extensions

Reply:
I added a sentence to the glossary